blockdag/subnetworks.go

```diff
@@ -85,66 +74,42 @@ func TxToSubnetworkID(tx *wire.MsgTx) (*subnetworkid.SubnetworkID, error) {
 	return subnetworkid.New(util.Hash160(txHash[:]))
 }
 
-// subnetwork returns a registered subnetwork. If the subnetwork does not exist
-// this method returns an error.
-func (s *SubnetworkStore) subnetwork(subnetworkID *subnetworkid.SubnetworkID) (*subnetwork, error) {
-	var sNet *subnetwork
-	var err error
-	dbErr := s.db.View(func(dbTx database.Tx) error {
-		sNet, err = dbGetSubnetwork(dbTx, subnetworkID)
-		return nil
-	})
-	if dbErr != nil {
-		return nil, errors.Errorf("could not retrieve subnetwork '%d': %s", subnetworkID, dbErr)
+// dbFetchSubnetwork returns a registered subnetwork.
+func dbFetchSubnetwork(subnetworkID *subnetworkid.SubnetworkID) (*subnetwork, error) {
```

Comment:
The "db" part is implementation details  - it shouldn't concern the caller from where is the subnetwork fetched from as long as he gets it.
Rename to .FetchSubnetwork(), or even go-er just .Subnetwork()